### ISSUE TITLE
Cordova Build Android not found assets files

### ISSUE THREAD

**Issue by jardimfelipe** [m0dch3n/vue-cli-plugin-cordova#131]
This is the first time that I installed cordova in a running Vue project. Everything works fine, but all images in /assets are not found in builded apk, but is found in cordova serve android command.

All assets are referenced like "../../assets" in this project, would this be a problem?

**Comment by JPilson** [comment]
Yes, I'm facing the same problem the workaround now is just to check if it's Android and then prefix the path with file:///android_asset/www/assets...

isAndroid?"file:///android_asset/www/assets/images/logo_around_clouds.svg"
